etc/profile-m-z/url-eater.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for url-eater
+# Description: Clean unnecessary parameters from URLs copied to clipboard
+# This file is overwritten after every install/update
+# Persistent local customizations
+include url-eater.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}/wayland-*
```

Comment:
Tested on Wayland?

Reply:
Yeah, didn't expect it would work when blacklisting that but it does. Tested several times in fact, rebooting/relogin, Xorg, Wayland, Sway and even in my old Compiz session the URL cleaning works just as advertised. Neat little app.

Reply:
Did you make sure it did not fallback to XWayland?

Reply:
Well, I used the example URL from the app's GitHub page in my Firefox (which definately runs on Wayland) and I copied that to several other apps, a mix of both Wayland and Xwayland apps. WFM, in both directions (copy from Xwayland app and paste in Wayland app and the other way around. I wouldn't mind taking the line out if you think that's safer, can always put it in my local's.

Reply:
I need to test the wayland clipboard interaction myself later.

Reply:
No rush. I'll flip this PR into draft mode, schedule another round of testing and hopefully we can figure this all out before the next firejail release. I had 'clipboard privacy/security' on my radar lately and in that context I stumbled on to url-eater. Will try to dig through its source code to get a better understanding of how it works. Thanks for following-up on this topic!

Reply:
My question is if you just block the wayland socket does it fallback to be an XWayland client and fail in Wayland-only or does it really work in Wayland w/o any (neither waland nor X11) socket.